### PR TITLE
Add documentation about windows

### PR DESCRIPTION
**Pull request by Smith-Cruise**
Add documentation about windows deploy.

### CODE REVIEW

**Comment by hermanzdosilovic** [comment]
I am sorry that we didn't defined what exactly you could do with this Windows compatibility.

Let's do that now :smile: 

1. You should change every compose file to use `postgres:9.6.5`.
2. Add note about Windows to remove lines from compose files that I mentioned in #38. Note that in `docker-compose.dev.yml` for Windows whole `nginx-proxy` service should be removed.

File: README.md [hunk 1]
```diff
@@ -11,57 +11,58 @@ on https://api.judge0.com.
 2. [Project Organization](#project-organization)
 3. [Quick Development Setup](#quick-development-setup)
 4. [Quick Production Setup](#quick-production-setup)
-5. [About Docker Images](#about-docker-images)
-6. [Adding New Compiler or Interpreter](#adding-new-compiler-or-interpreter)
-7. [HTTPS In Production](#https-in-production)
-8. [Notes](#notes)
+5. [Carefully in Windows](#carefully-in-windows)
```
Comment (hermanzdosilovic) [inline README.md]: I would add this section as an subsection of `Notes`. Also I would just name this `Running on Windows`.

File: README.md [hunk 2]
```diff
@@ -11,57 +11,58 @@ on https://api.judge0.com.
...
-|35|Python (3.5.3)|
-|36|Python (2.7.9)|
-|37|Python (2.6.9)|
-|38|Ruby (2.4.0)|
-|39|Ruby (2.3.3)|
-|40|Ruby (2.2.6)|
-|41|Ruby (2.1.9)|
-|42|Rust (1.20.0)|
-|43|Text (plain text)|
+|  #   |                 Name                 |
```
Comment (hermanzdosilovic) [inline README.md]: You don't need to do that. Spaces do not matter for Markdown. It is also easier to maintain whole table if there is no alignment in Markdown source.

File: README.md [hunk 3]
```diff
@@ -117,7 +118,9 @@ To host your own Judge0 API you need to install [Docker](https://docs.docker.com
     ```
     $ wget https://raw.githubusercontent.com/judge0/api/master/judge0-api.conf.default -O judge0-api.conf
     ```
+
```
Comment (hermanzdosilovic) [inline README.md]: Same comment as for the table above. Please remove these new lines.

File: README.md [hunk 4]
```diff
@@ -117,7 +118,9 @@ To host your own Judge0 API you need to install [Docker](https://docs.docker.com
...
 3. In `judge0-api.conf` change `RAILS_ENV` to `production`.
+
```
Comment (hermanzdosilovic) [inline README.md]: Same comment as for the table above. Please remove these new lines.

File: README.md [hunk 5]
```diff
@@ -130,7 +133,42 @@ To host your own Judge0 API you need to install [Docker](https://docs.docker.com
...
+    env_file: judge0-api.conf
+    restart: always
+
+  redis:
+    image: redis
+    restart: always
+```
+
 ## About Docker Images
+
```
Comment (hermanzdosilovic) [inline README.md]: Same comment as for the table above. Please remove these new lines.

File: README.md [hunk 6]
```diff
@@ -130,7 +133,42 @@ To host your own Judge0 API you need to install [Docker](https://docs.docker.com
 
 6. Open `http://<IP OF YOUR SERVER>:3000` in your browser.
 
+## Carefully in Windows
+
+If you want to run Judge0 API on Windows, you have to change file `docker-compose.yml` .
```
Comment (hermanzdosilovic) [inline README.md]: There is no `docker-compose.yml`. There are: `docker-compose.prod.yml`, `docker-compose.prod.https.yml` and `docker-compose.dev.yml`. If on Windows you need to change compose file you are using.

File: README.md [hunk 7]
```diff
@@ -130,7 +133,42 @@ To host your own Judge0 API you need to install [Docker](https://docs.docker.com
...
+
+This is sample about `docker-compose.yml` in Windows.  We had run it succeed on Windows10 via `PowerShell`.
```
Comment (hermanzdosilovic) [inline README.md]: You could create new `docker-compose.windows.yml` in repo and just reference to it. I like the part about successful run on Windows 10 :smile:, but PowerShell comment is not necessary if you can also run it via good old CMD.

File: README.md (hunk 1 above)
Comment (Smith-Cruise) [reply README.md]: Maybe someone will not notice it

File: README.md (hunk 2 above)
Comment (Smith-Cruise) [reply README.md]: oka, sorry, that's not my fault, my editor do it automatically.

**Pull request closed without merging**